pkgs/desktops/budgie/budgie-control-center/default.nix

```diff
@@ -158,7 +162,7 @@ stdenv.mkDerivation rec {
       --prefix XDG_DATA_DIRS : "${gdk-pixbuf}/share"
       --prefix XDG_DATA_DIRS : "${librsvg}/share"
       # WM keyboard shortcuts
-      --prefix XDG_DATA_DIRS : "${gnome.mutter}/share"
+      --prefix XDG_DATA_DIRS : "${magpie}/share"
```

Comment:
I think using mutter for this is fine, anyway our magpie does not provide keyboard shortcuts right now https://github.com/NixOS/nixpkgs/commit/31c1161b085f151234812086269881b0adeb7257 :-)